src/dev/code_coverage/ingest_coverage/index_mapping.md

```diff
@@ -0,0 +1,194 @@
+# Create index mapping 
+
+This is usually done in Kibana's dev tools ui.
```

Comment:
Should there be additional information here? When would someone need these mappings?

Reply:
Well, that's really just a place to hold the code until we figure out how to move saved objects around efficiently.  But, if we lose a cluster, we can recover from here.